Milyli.ScriptRunner.Core/Tools/RelativityScriptProcessor.cs

```diff
@@ -84,6 +84,10 @@ public string SubstituteGlobalVariables(int workspaceId, string inputSql)
 		/// <inheritdoc />
 		public string SubstituteSavedSearchTables(IEnumerable<JobScriptInput> populatedInputs, IEnumerable<RelativityScriptInputDetails> relativityScriptInputDetails, IDictionary<int, string> searchTablePairs, string inputSql)
 		{
+			// Optional input  are not handled explicitly for saved searches --
+			// Per relativity this is not officially supported, so if an optional search input
+			// is not supplied the script execution will simply fail.
+			// https://devhelp.relativity.com/t/documentation-around-non-required-relativity-script-inputs/6535/5

```

Comment:
Is there a way we can hard fail here with a clear error message? Or is that actually worse than failing like Relativity would?

Reply:
I'd rather just let it fall through to the error you'd get with the script since the default behavior is "it's just null, even if it's really an unescaped string substituted for a column name" (not sure how that works, but it does), except in the case of saved searches where it just flat out doesn't work anyway.